src/translate/expr.rs

```diff
@@ -16,10 +18,8 @@ use {
 pub fn translate_expr(sql_expr: &SqlExpr) -> Result<Expr> {
     match sql_expr {
         SqlExpr::Identifier(ident) => match ident.quote_style {
-            Some(_) => Ok(Expr::Literal(AstLiteral::QuotedString(
-                ident.value.to_owned(),
-            ))),
-            None => Ok(Expr::Identifier(ident.value.to_owned())),
+            Some(_) => Ok(Expr::Literal(AstLiteral::QuotedString(ident.value.clone()))),
```

Comment:
Wow, that's really good catch to replace `ident.value.to_owned()` to `ident.value.clone()`!
`ident.value` type is `String` so.. `clone` is better in here. Thanks!